docs/SPEC.md

```diff
@@ -1218,6 +1218,21 @@ Example:
 
 **Note:** The `yield` function produces side effects.
 
+#### Fill
+
+Fill will scan a stream for nil values and replace them with a non-null value.  
```

Comment:
s/non-null/non-nil/?

Reply:
the rest of the spec uses `null`.  I'm ok with changing it, but it'd need to change everywhere.

Reply:
...except for this PR, where I use one of each for every other sentence.   Fixed all instances to null